modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgnitePdsAtomicCacheHistoricalRebalancingTest.java

```diff
@@ -61,4 +63,11 @@
         if (!walRebalanceInvoked)
             throw new AssertionError("WAL rebalance hasn't been invoked.");
     }
+
+    /** {@inheritDoc} */
+    @Test
+    @Ignore
+    @Override public void testTopologyChangesWithConstantLoadOnInMemoryCache() {
+        // No-op.
+    }
```

Comment:
I think it is not good for adding new ignored tests. Why we should do so?

Reply:
Because testTopologyChangesWithConstantLoad is the best that I found to check complex rebalancing with indexing, but as you may know, historical (WAL) rebalancing doesn't support for in-memory caches, so test case with in-memory cache should be ignored somehow.

Reply:
I've updated PR, added a new test for mixed configurations of the data regions.